lib/active_admin/view_helpers/display_helper.rb

```diff
@@ -25,13 +25,7 @@ def display_name_method_for(resource)
         @@display_name_methods_cache ||= {}
         @@display_name_methods_cache[resource.class] ||= begin
           methods = active_admin_application.display_name_methods - association_methods_for(resource)
-          method  = methods.detect{ |method| resource.respond_to? method }
-
-          if method != :to_s || resource.method(method).source_location
-            method
-          else
-            DISPLAY_NAME_FALLBACK
-          end
```

Comment:
I think removing this code results in the fact that `DISPLAY_NAME_FALLBACK` is never been used, because of every object (except BasicObject / BasicObject Subclasses) respond to `:to_s`.

```
irb(main):001:0> Class.new.new.respond_to? :to_s
=> true
irb(main):002:0> Class.new.new.method(:to_s).source_location
=> nil
```